### PR TITLE
 Change in Project Manager

### PR DESCRIPTION
**Pull request by GibionMillak**
I made a small change in the project manager, make the interface a little more similar to the rest of the project. 
I'm taking a "look" at the interface and seeing what could be improved.

before

after

### CODE REVIEW

**Comment by AThousandShips** [comment]
The buttons now doesn't have text so how are you supposed to know what they do?

File: editor/project_manager.cpp [hunk 1]
```diff
@@ -2710,207 +2712,141 @@ ProjectManager::ProjectManager() {
...
-			language_btn->add_item(vformat("[%s] %s", lang, lang_name), i);
-			language_btn->set_item_metadata(i, lang);
-			if (current_lang == lang) {
-				language_btn->select(i);
-			}
+	language_btn->set_text(current_lang);
+	for (int i = 0; i < editor_languages.size(); i++) {
+		String lang = editor_languages[i];
+		String lang_name = TranslationServer::get_singleton()->get_locale_name(lang);
+		language_btn->add_item(vformat("%s [%s]", lang_name, lang), i);
```
Comment (AThousandShips) [inline editor/project_manager.cpp:2825]: Why this change?

**Comment by AThousandShips** [comment]
Why have you removed the version?

**Comment by GibionMillak** [comment]
> The buttons now doesn't have text so how are you supposed to know what they do?

buttons

All I tried to do was reassign the buttons to the same style as the general interface... And like all other buttons they have tooltips.
90% of the engine's buttons have no text.

**Comment by GibionMillak** [comment]
> Why have you removed the version? Also that and probably other controls are now left unused

I tried to make it more "clean", since the version information is still there. It was repetitive information.
version

**Comment by AThousandShips** [comment]
Most of them do not have tooltips thoug? And the version is convenient to not have to go to the help window, just like it's visible in the editor

**Comment by AThousandShips** [comment]
You have left a ton of functionality and items that now do nothing though, you need to do a lot of cleaning up and can't just leave things dangling

**Comment by GibionMillak** [comment]
> You have left a ton of functionality and items that now do nothing though,

For example?

**Comment by AThousandShips** [comment]
The version button and related functionality

**Comment by AThousandShips** [comment]
My minimal suggestion is:
* Leave version and language where they are
* Have the search expand

But I think that putting the icons on a single row makes it harder to get an overview, and it makes the interface more crowded, it also makes it so you can't make the window as small and compact, and makes the search bar more crowded on small windows

**Comment by jcostello** [comment]
I like the changes. I would suggest you to do a proposal to explain all your changes so they can be consider for next versions

**Comment by AThousandShips** [comment]
@jcostello please don't comment to boost

**Comment by jcostello** [comment]
Boost? This was active an hour ago. If you re-read my comment I just pointed out that I like the changes and suggest him to write a proposal

**Comment by AThousandShips** [comment]
You didn't add anything to the PR, that's what I mean by boosting

**Comment by Calinou** [comment]
Unlabeled icon buttons should be avoided when possible, as it's hard to figure out what they'll do unless you click them.

**Comment by dalexeev** [comment]
We could leave labels on important buttons and hide on rarely used ones.

**Comment by AThousandShips** [comment]
Also you have made changes to the asset library viewer but you haven't shown how it looks now

**Comment by seppoday** [comment]
Hello, I've just created quick mockup to give to some fundamentals to my opinion about this PR.

I totally agree that it should be proposal and in my opinion PR mockup kinda lacks key points why some things changed. In my opinion making it more like a menu in editor itself is not a great idea. Calinou explained it well.

So what I've changed and how it compares to this PR.

1. Moved buttons to top row. It is kinda good idea but I think labels are necesary and OP tried to make it more clean. With all buttons it is not that clear and few of them are buttons "embeded" to project.
2. I moved some buttons to contextual menu with 3 dots next to project itself. You can run project by clicking on it or clickign 3 dots > Run.
3. With more horizontal space I've added 2 columns. "Godot version" and "last modified".
4. I've moved "About" button next to software version. It is kinda connected in my opinion.
5. I've also added headers to project list to give more context.
6. Few small visual changed here and there. For example moving language to same line as sorting (to save space)

**I creating this mockup really quick and it is not my proposition. Just wanted to say that there is alot of things that we can change but we need proposal, discussion and probably a lot of mockup propositions.**

obraz

**Comment by seppoday** [comment]
> Just to note that this kind of specific version info is not a feature in the current version, and not at all supported for pre 4.0, but other than that this looks much cleaner and less cluttered
> 
> I think the language should remain where it is to not take up as much space in the bar, as the size of the bar really controls how narrow the window can be, which is a thing that really needs to be considered
> 
> Also I think the loss of an easily accessible "run" button is not a good way to go, in general hiding basic functionality behind a context menu makes the interface far less easy to navigate
> 
> I honestly think the project manager is very good as it is however, and as it is the very first thing someone new sees making it complicated is not necessarily helpful

Yea sure you can go in that direction. 

It is not proposal from my side, just wanted to coment on that PR that it should be discussed and forst you need to find problems to fix them later not another way around. My mockups might not sell that idea too well. Sorry about that. Nonethless I also don't really have much of a problem with existing project manager. It might be improved a little but I never felt that way before checking this PR.
obraz

**Comment by Ansraer** [comment]
> Sorry I have a problem with creating stuff that noone was asking for...

Don't worry about it @seppoday, you can never have too many mockups. I am not a UI/UX guy, but the few times I tried to build a user interface without having someone else design it the result was barely runctional.
Since OP has yet to create a proposal maybe you could do so? I really think you are on to something with your mockups and a proposal would give them more exporsure and feedback. Especially if you are willing to do a few more iterations.

Btw, some feedback for your last iteration: I like that the "Create new" button is more visible, but you might have overdone it a bit 😁 . Maybe you could come up with something in between?
I also don't think we need the edit buttons since I would expect double clicks on the project name to open it. And I am also not sure if the play button to be this visible. Maybe male it a bit more subtle? In my experience you rarely need it.
